q2_humann3/plugin_setup.py

```diff
@@ -93,7 +94,24 @@
         "bowtie_database": "directory containing the bowtie2 database reference files",
     },
     parameter_descriptions={
-        "threads": "number of threads/processes",
+        "sample_threads": (
```

Comment:
I feel like sample_threads isn't a great name for this parameter because it's used as the size of the process pool used to process the sequences. I would suggest something like "worker_pool_size" or "sample_process_count" or maybe even like "parallel_sample_limit". 

Otherwise,  PR functionality looks great.

Reply:
@matt-hudgins I want to be explicit that this parameter sets the number of samples that should be run in parallel, rather than the number of workers that can run on a single sample. what about `n_parallel_samples`?